frontend/benefit/applicant/src/components/applications/forms/application/step5/ApplicationFormStep5.tsx

```diff
@@ -1,16 +1,88 @@
+import { DynamicFormStepComponentProps } from 'benefit/applicant/types/common';
+import {
+  Button,
+  IconArrowRight,
+  IconDocument,
+  IconPenLine,
+  IconPrinter,
+} from 'hds-react';
 import * as React from 'react';
-import FormSection from 'shared/components/forms/section/FormSection';
+import {
+  $Grid,
+  $GridCell,
+  $Hr,
+} from 'shared/components/forms/section/FormSection.sc';
 
-const ApplicationFormStep5: React.FC = () => {
-  const { t } = useTranslation();
-  const translationsBase = 'common:applications.sections.credentials';
+import StepperActions from '../stepperActions/StepperActions';
+import CredentialsSection from './credentialsSection/CredentialsSection';
+import { useApplicationFormStep5 } from './useApplicationFormStep5';
 
+const ApplicationFormStep5: React.FC<DynamicFormStepComponentProps> = ({
+  data,
+}) => {
+  const { t, handleBack, handleNext, translationsBase } =
+    useApplicationFormStep5(data);
   return (
     <>
-      <FormSection header={t(`${translationsBase}.heading1`)}>
-        Content
-      </FormSection>
+      <CredentialsIngress />
+      <$Grid>
+        <$GridCell $colSpan={6}>
+          <CredentialsSection
+            title={t(`${translationsBase}.electronicPowerOfAttorney.title`)}
+            description={t(
+              `${translationsBase}.electronicPowerOfAttorney.description`
+            )}
+            icon={<IconPenLine size="l" />}
+            actions={
+              <Button
+                theme="black"
+                variant="secondary"
+                iconRight={<IconArrowRight />}
+              >
+                {t(`${translationsBase}.electronicPowerOfAttorney.action1`)}
+              </Button>
+            }
+          />
+        </$GridCell>
+        <$GridCell $colSpan={6}>
+          <CredentialsSection
+            title={t(`${translationsBase}.uploadPowerOfAttorney.title`)}
+            description={t(
+              `${translationsBase}.uploadPowerOfAttorney.description`
+            )}
+            icon={<IconDocument size="l" />}
+            actions={
+              <$Grid>
+                <$GridCell $colSpan={4}>
+                  <Button
+                    theme="black"
+                    variant="secondary"
+                    iconLeft={<IconPrinter />}
+                  >
+                    {t(`${translationsBase}.uploadPowerOfAttorney.action1`)}
+                  </Button>
+                </$GridCell>
+                <$GridCell $colSpan={6}>
+                  <Button
+                    theme="black"
+                    variant="secondary"
+                    iconLeft={<IconArrowRight />}
+                  >
+                    {t(`${translationsBase}.uploadPowerOfAttorney.action2`)}
+                  </Button>
+                </$GridCell>
+              </$Grid>
+            }
+          />
+        </$GridCell>
+      </$Grid>
+      <$Hr $hasMarginBottom />
```

Comment:
For controlling margins, I find the `css` prop from `styled-components` useful to avoid implementing new components or adjustments to cover all the cases.

```suggestion
     cosnt theme = useTheme();
     ...
      <$Hr css={`
          margin-bottom: ${theme.spacing.l};
       `}
      />
```TSX